Add pep8 and cleaning up info messages and UI

tin2tin opened the pull request:
I ran the code through a pep8 check, used Titel case for info messages(added a few more of those), and tried to rename the buttons so they are a bit more clear what they do. 

I hope you like these changes. 

It would be nice with some better progress info messages during the operation, but that's over my level. :-)

tin2tin committed:
Cleaning up info messages and UI

cgvirus commented on VSE_Easy_Proxy.py:
```diff
@@ -93,265 +82,296 @@ class ToggleEasyProxy(bpy.types.Operator):
     bl_idname = "sequencer.create_easy_all_proxy"
-    bl_label = "Create All Proxy"
-    bl_description = "Create all proxy with ffmpeg"
+    bl_label = "Build for All Strips"
```
Thanks for the contrib.
Can you change it to "Proxy All Strips" ? 
max 3 word button is manageable for space reason.

cgvirus commented:
Thanks for the contribution. I think, I better test it before merging. Will let you know.

tin2tin committed:
Update VSE_Easy_Proxy.py

tin2tin committed:
Update README.md

cgvirus commented:
Closed issued by Overlord.
Reason: 
conflict with future maps.

tin2tin commented:
I'm sorry, I didn't know that when I updated my repro, this patch was also updated, that was not my intention. Anyhow, I should just have closed this patch a long time ago. True that the pep8 made it very hard to see the actual changes.
